stdlib/messaging/activemq-artemis/src/main/ballerina/artemis/artemis_commons.bal

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+
+# Constant for the artemis error code.
+public const string ARTEMIS_ERROR_CODE = "{ballerina/artemis}ArtemisError";
+
+# The Artemis error record.
+#
+# + message - the error message.
+public type ArtemisError record {
+    string message = "";
```

Comment:
Shall we make this an optional field if the message is not mandatory instead of setting a default value?